### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'confPath')

### ISSUE THREAD

**Issue by manojsitapara** [open-wa/wa-automate-nodejs#2748]
When run docker container, getting following error. it was working fine till yesterday.

> 
> Run `npm audit` for details.
> npm notice 
> npm notice New patch version of npm available! 8.12.1 -> 8.12.2
> npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.12.2>
> npm notice Run `npm install -g npm@8.12.2` to update!
> npm notice 
> TypeError: Cannot read properties of undefined (reading 'confPath')
>     at configFile (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/setup.js:159:55)
>     at cli (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/setup.js:194:108)
>     at /usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:41:76
>     at Generator.next (<anonymous>)
>     at /usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:8:71
>     at new Promise (<anonymous>)
>     at __awaiter (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:4:12)
>     at start (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:40:12)
>     at Object.<anonymous> (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:224:1)
>     at Module._compile (node:internal/modules/cjs/loader:1112:14)

**Comment by abdulaziz-git** [comment]
I'm also got same error when running on Ubuntu 20.04 with nodejs v14x

> TypeError: Cannot read properties of undefined (reading 'confPath')
    at configFile (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/setup.js:159:55)
    at cli (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/setup.js:194:108)
    at /usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:41:76
    at Generator.next (<anonymous>)
    at /usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:4:12)
    at start (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:40:12)
    at Object.<anonymous> (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/index.js:224:1)
    at Module._compile (node:internal/modules/cjs/loader:1112:14)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.40.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.40.1...4.40.2) (2022-06-17)

- 🚑  fix #2748 [`#2748`](https://github.com/open-wa/wa-automate-nodejs/issues/2748)
- Release 4.40.2 [`530c18d`](https://github.com/open-wa/wa-automate-nodejs/commit/530c18db51111d7d8c62f35a58ed3c1834b76fd5)
- updated types-only package [`63c9417`](https://github.com/open-wa/wa-automate-nodejs/commit/63c941744e851799294ad0fec586218488a1ab56)

**Comment by abdulaziz-git** [comment]
@smashah is working now, thanks
